Index: audio.c
===================================================================
--- audio.c
+++ audio.c	2010-04-01 21:42:38.000000000 +0200
@@ -1736,7 +1736,7 @@
 {
   if (message)
     {
-      mus_print(message);
+      mus_print(message, "%s");
       free(message);
     }
   return(MUS_ERROR);
Index: sndplay.c
===================================================================
--- sndplay.c
+++ sndplay.c	2010-04-01 21:45:59.000000000 +0200
@@ -294,7 +294,7 @@
     {
       if (strcmp(argv[i], "-describe") == 0)
 	{
-	  fprintf(stdout, mus_audio_describe()); 
+	  fprintf(stdout, mus_audio_describe(), "%s"); 
 	  exit(0);
 	}
       else 
Index: sndinfo.c
===================================================================
--- sndinfo.c
+++ sndinfo.c	2010-04-01 21:52:16.000000000 +0200
@@ -105,7 +105,7 @@
 	  if ((chans > 0) && (mus_sound_maxamp_exists(argv[ctr])))
 	    {
 	      ampstr = display_maxamps(argv[ctr], chans);
-	      if (ampstr) fprintf(stdout, ampstr);
+	      if (ampstr) fprintf(stdout, ampstr, "%s");
 	    }
 	  fprintf(stdout, "\n");
 	  if (comment) fprintf(stdout, "  comment: %s\n", comment);
