diff --git a/snd-gfile.c b/snd-gfile.c
index 3670731..f8b2634 100644
--- a/snd-gfile.c
+++ b/snd-gfile.c
@@ -4310,7 +4310,7 @@ void save_edits_now(snd_info *sp)
   dialog = unsaved_edits_dialog(sp);
   if (!dialog)
     {
-      dialog = gtk_message_dialog_new(NULL, GTK_DIALOG_MODAL, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO, (const gchar *)question);
+      dialog = gtk_message_dialog_new(NULL, GTK_DIALOG_MODAL, GTK_MESSAGE_QUESTION, GTK_BUTTONS_YES_NO,"%s", (const gchar *)question);
       SG_SIGNAL_CONNECT(dialog, "response", unsaved_edits_activate, (gpointer)sp);
       save_unsaved_edits_dialog(dialog, sp);
     }
